setup.py

```diff
@@ -76,8 +76,8 @@ def collectExtraFiles():
         "yamlize==0.7.1",
     ],
     extras_require={
-        "mpi": ["mpi4py==3.0.3"],
```

Comment:
Removing this version pin should allow the right max versions to get installed for tests. 

Testing internally to make sure this doesn't mess anything up!

Reply:
Ok this is OK to remove, resolving